src/aws_encryption_sdk/streaming_client.py

```diff
@@ -83,28 +80,27 @@ class _ClientConfig(object):
     """
 
     source = attr.ib(hash=True, converter=aws_encryption_sdk.internal.utils.prep_stream_data)
-    materials_manager = attr.ib(
-        hash=True, default=None, validator=attr.validators.optional(attr.validators.instance_of(CryptoMaterialsManager))
-    )
-    key_provider = attr.ib(
-        hash=True, default=None, validator=attr.validators.optional(attr.validators.instance_of(MasterKeyProvider))
-    )
-    source_length = attr.ib(
-        hash=True, default=None, validator=attr.validators.optional(attr.validators.instance_of(six.integer_types))
-    )
+    materials_manager = attr.ib(hash=True, default=None, validator=optional(instance_of(CryptoMaterialsManager)))
```

Comment:
Why is this changing?  It looks like it is just formating?

Reply:
I changed the validator references from their fully-qualified names to importing the validators directly to reduce boilerplate.